etc/scripts/redfish-update.in

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+action="$1"
+interface="$2"
+
+case $action in
+post-up)
+	@wicked_sbindir@/wicked redfish hosts-update || :
```

Comment:
Is there a reason, why we hide errors?

Reply:
We don't need to handle them multiple times or trigger further error log than `wicked redfish` logs to syslog already.